service/internal/logger/logger.go

```diff
@@ -17,10 +18,34 @@ type Config struct {
 	Type   string `yaml:"type" default:"json"`
 }
 
-func (l *Logger) With(key string, value string) *Logger {
-	return &Logger{
-		Logger: l.Logger.With(key, value),
+// From the Slog docs (https://betterstack.com/community/guides/logging/logging-in-go/#customizing-slog-levels):
+// The log/slog package provides four log levels by default, with each one
+// associated with an integer value: DEBUG (-4), INFO (0), WARN (4), and ERROR (8).
+const (
+	// Currently setting AUDIT level to 6, between WARN and ERROR
+	LevelAudit = slog.Level(6)
```

Comment:
@strantalis, wondering what would be more appropriate here. I'm thinking maybe between INFO and WARN might be better (2)?

Reply:
Can we make it higher than error and not allow them to turn it off?

Reply:
Yeah I made it level 10, and the way the logger is initialized the `getLevel` function only accepts the options "debug", "info" and "error" and will throw an error if anything else is provided as the level, so in that way it would never be possible to turn off